src/graphql/schema.js

```diff
@@ -66,6 +66,14 @@ type PageInfo {
   hasNextPage: Boolean!
   pageNumber: Int!
   count: Int!
+  nextSkips: NextSkips
```

Comment:
lets call this `nextTransactionSkip`. dont think this will need to be used by another query.